code/modules/mob/living/carbon/human/human.dm

```diff
@@ -111,6 +111,8 @@
 		if(!HAS_TRAIT(H, TRAIT_SECURITY_HUD) && !HAS_TRAIT(H, TRAIT_MEDICAL_HUD))
 			return
 		var/datum/data/record/R = find_record("name", perpname, GLOB.data_core.general)
```

Comment:
You'd have to change the `R` everywhere else under here, but since you're already there, might as well use good coding standards.
```suggestion
		var/datum/data/record/general_record = find_record("name", perpname, GLOB.data_core.general)
```